todo/src/components/ToDo.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { toggleTodo } from '../actions/toDoActions'
+import { useSelector, useDispatch } from 'react-redux'
+
+const ToDo = () => {
+    const dispatch = useDispatch()
+
+    const items = useSelector(state => {
+        return Object.values(state.toDos)
+    })
+    return (
+        <div>
+            <ul>
+            <h2>TO DO LIST</h2>
+            {items.map((item) => {
+                return item.completed ? (<li style={{textDecoration: "line-through"}} key={item.id} onClick={() => dispatch(toggleTodo(item.id))}>{item.text}</li>) : (<li  key={item.id} onClick={() => dispatch(toggleTodo(item.id))}>{item.text}</li>)
```

Comment:
Instead of wrapping the whole thing in the ternary, you can only wrap the style section with your ternary

Reply:
`style={      items.completed ? {textDecoration: "line-through"} : "" }`